src/Microsoft.AspNetCore.SpaTemplates/content/Aurelia-CSharp/Properties/launchSettings.json

```diff
@@ -0,0 +1,41 @@
+{
+  "iisSettings": {
+//#if (WindowsAuth)
```

Comment:
There isn't a WindowsAuth option for any of the SPA templates. Just to avoid confusing anyone, could we remove these if/else blocks from the templates where they don't apply?

Reply:
Does this also apply to RequireHttps?

Reply:
What is RequireHttps? Is that an existing concept? I don't see it in the existing `template.json` files.

Reply:
https://github.com/aspnet/templating/blob/dev/src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/.template.config/template.json#L286-L289. Does the Spa templates have any organizational or individual auth? If so I can add this in here.

Reply:
No, there's no auth option in the SPA templates yet.